zebrad/src/components/mempool/crawler.rs

```diff
@@ -106,6 +119,36 @@ where
             transaction_ids.len()
         );
 
-        // TODO: Send transaction IDs to the download and verify stream (#2650)
+        if !transaction_ids.is_empty() {
+            if let Err(error) = self.queue_transactions(transaction_ids).await {
+                error!(
+                    "Failed to forward crawled transactions to mempool transaction downloader: {}",
+                    error
+                );
+            }
+        }
+    }
+
+    /// Forward the crawled transactions IDs to the mempool transaction downloader.
+    async fn queue_transactions(&mut self, transaction_ids: Vec<Gossip>) -> Result<(), BoxError> {
+        let response = self
+            .mempool
+            .ready_and()
+            .await?
+            .call(mempool::Request::Queue(transaction_ids))
+            .await?;
+
+        let queue_errors = match response {
+            mempool::Response::Queued(queue_results) => {
+                queue_results.into_iter().filter_map(Result::err)
+            }
+            _ => unreachable!("Mempool did not respond with queue results to mempool crawler"),
+        };
+
+        for error in queue_errors {
+            warn!("Failed to download a crawled transaction: {}", error);
+        }
```

Comment:
I'm not sure if this error is particularly serious, or worth reporting at the default `info` log level.

Failing to download a crawled transaction happens regularly, because:
* the crawler fans out to ~4 peers
* the downloader tries a random peer that has recently gossiped the transaction, or any random peer

```suggestion
        for error in queue_errors {
            debug!("Failed to download a crawled transaction: {}", error);
        }
```